README.md

```diff
@@ -92,16 +92,16 @@ The syntax for the _sort_by_ parameter is as follows:
 
 #### Filtering and Sorting by Association attribute
 
-Requests can also be filtered by assocation attribute and sorted by association attribute and count in addition to the direct attribute specified as in the above examples.
+Requests can also be filtered by assocation attribute and sorted by association attribute and count in addition to the direct attribute specified as in the above examples. When specifying attribute associations, the filter operators must be specified.
 
 Single level association can be specified as follows:
 
 ##### Filter by association attribute:
 
 | Query Parameter | Ruby Client Parameter | GraphQL Parameter |
 | --------------- | --------------------- | ----------------- |
-| "?filter[association][name]=reviews" | { :filter => { :association => { :name => "reviews" } } } | filter: { association: { name: "reviews" } } |
 | "?filter[association][name][eq]=reviews" | { :filter => { :association => { :name => { :eq => "reviews" } } } } | filter: { association: { name: { eq: "reviews" } } } |
+| "?filter[source_type][name][eq][]=OpenStack&filter[source_type][name][eq][]=OpenShift" | { :filter => { :source_type => { :name => { :eq => [ "OpenStack", "OpenShift" ] } } } } | filter: { source_type: { name: { eq: [ "OpenStack", "OpenShift" ] } } } |
```

Comment:
So I get how this is working - why is there an extra `[]` after `[eq]`? I thought that eq was the operator specifier and that would be enough.

Reply:
The `[]`is how you specify an array.  In this case "OpenStack" and "OpenShift".

Reply:
Ahhhhh ok, that makes more sense. that way it ends up with this `filter: { source_type: { name: { eq: [ "OpenStack", "OpenShift" ] } }`

Thanks for the clarification!